pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/errorprone/xml/UnnecessaryCaseChange.xml

```diff
@@ -54,7 +54,7 @@ public class Foo {
 
     <test-code>
         <description>failure case with toLowerCase().equals()</description>
-        <expected-problems>1</expected-problems>
+        <expected-problems>2</expected-problems>
```

Comment:
I don't think, we should report here two violations, we have only one call to equals...